src/ng-select/ng-select.component.ts

```diff
@@ -733,14 +733,22 @@ export class NgSelectComponent implements OnDestroy, OnChanges, AfterViewInit, C
         if (!this.isOpen || !this.dropdownPanel) {
             return;
         }
-        this.dropdownPanel.scrollInto(this.itemsList.markedItem);
+        this.dropdownPanel.scrollTo(this.itemsList.markedItem);
     }
 
     private _scrollToTag() {
         if (!this.isOpen || !this.dropdownPanel) {
             return;
         }
-        this.dropdownPanel.scrollIntoTag();
+        this.dropdownPanel.scrollToTag();
+    }
+
+    private _onSelectionChanged() {
+        if (this.isOpen && this.multiple && this.appendTo) {
+            // Make sure items are rendered.
+            this._cd.detectChanges();
+            this.dropdownPanel.adjustDropdownPosition();
```

Comment:
As this is already dropdownPanel maybe it make sense to drop dropdown name from method.

```ts
this.dropdownPanel.adjustPosition()
```